AdobeIms/Test/Unit/Model/ConfigTest.php

```diff
@@ -8,22 +8,31 @@
 namespace Magento\AdobeIms\Test\Unit\Model;
 
 use Magento\AdobeIms\Model\Config;
+use Magento\Config\Model\Config\Backend\Admin\Custom;
 use Magento\Framework\App\Config\ScopeConfigInterface;
-use Magento\Framework\TestFramework\Unit\Helper\ObjectManager;
+use Magento\Framework\UrlInterface;
 use PHPUnit\Framework\MockObject\MockObject;
 use PHPUnit\Framework\TestCase;
 
 /**
- * Config data test.
+ * PHPUnit test for \Magento\AdobeIms\Model\Config
  */
 class ConfigTest extends TestCase
 {
-    private const CONFIG_XML_PATH_API_KEY = 'adobe_ims/integration/api_key';
-
     /**
-     * @var ObjectManager
+     * Constants for test cases
      */
-    private $objectManager;
+    private const API_KEY = 'API_KEY';
+    private const LOCALE_CODE = 'en_US';
+    private const PRIVATE_KEY = 'PRIVATE_KEY';
+    private const REDIRECT_URI = 'REDIRECT_URI';
+    private const ACCCESS_TOKEN = 'ACCCESS_TOKEN';
+    private const TOKEN_URL = 'https://token-url.com/integration';
+    private const IMAGE_URL_DEFAULT = 'https://image-url.com/default';
+    private const CALLBACK_URL = 'https://magento-instance.com/adobe_ims/oauth/callback';
+    private const IMAGE_URL_PATTERN = 'https://image-url.com/pattern?api_key=#{api_key}';
+    private const LOGOUT_URL_PATTERN = 'https://logout-url.com/pattern?access_token=#{access_token}&redirect_uri=#{redirect_uri}';
+    private const AUTH_URL_PATTERN = 'https://auth-url.com/pattern?client_id=#{client_id}&redirect_uri=#{redirect_uri}&locale=#{locale}';
```

Comment:
Hi @hws47a, any suggestions on how to split this string constant into a multiline?